src/views/home/components/CopyEditionModal.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+    <b-modal
+        v-if="edition"
+        id="copy-edition-modal"
+        ref="copyModalRef"
+        header-class="title-header"
+        footer-class="title-footer"
+        @shown="copyModalShown"
+        @ok="copyEdition"
+        :ok-disabled="waiting || !canCopy"
+        :cancel-disabled="waiting"
+    >
+        <template v-slot:modal-header>
+            <b-row class="mt-3">
+                <b-col cols="12">
+                    {{
+                        $t('home.copyTitle', {
+                            name: edition.name,
+                            owner: edition.owner.forename,
+                        })
+                    }}</b-col
+                >
+            </b-row>
+        </template>
+        <form @submit.stop.prevent="copyEdition">
+            <b-form-input
+                ref="newCopyName"
+                id="newName"
+                v-model="newCopyName"
+                type="text"
+                @keyup.enter="copyEdition"
+                required
+                :placeholder="$t('home.newEditionName')"
+            ></b-form-input>
```

Comment:
I know it isn't on the specs, but it might be nice to include here the fields for the copyrightHolder and collaborators.  The copyrightHolder field can be prefilled with the full name of the current user.  The collaborators can be left blank (it is automatically populated from the edition editors if it is left blank).  Both those values are part of theJSON sent to the `POST /v1/editions/{editionId}` endpoint.

Reply:
Who is going to fill these fields? The backend or should we fill them when creating the clone? I Think this should be the backend's responsibility.

Reply:
The user can choose to specify a different copyrightHolder than themselves (for instance the institution/project they work for), if no value is submitted to the API for this field, the backend assumes that the user making the request will be the copyright holder.  The user may also explicitly specify collaborators, or if left blank the backend will always autogenerate a list of collaborators based on the editors working on the edition.